src/as-validator-issue-tag.h

```diff
@@ -921,6 +921,26 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("This color is not a valid HTML color code."),
 	},
 
+	{ "custom-invalid-tag",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The custom tags can only contains value tags"),
+	},
+
+	{ "custom-key-missing",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("This custom tag is missing the 'key attribute"),
+	},
+
+	{ "custom-key-duplicated",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("A key can only be used once"),
+	},
+
+	{ "custom-value-missing",
+	  AS_ISSUE_SEVERITY_ERROR,
```

Comment:
Unless this causes problems, making this a WARNING or possibly even INFO might be better. That allows some flexibility for writers of custom data (e.g. to have placeholders that are ignored deliberately).